sql/item_cmpfunc.h

```diff
@@ -303,11 +303,11 @@ class Item_func_isnottrue : public Item_func_truth
   Item_func_isnottrue(THD *thd, Item *a):
     Item_func_truth(thd, a, true, false) {}
   ~Item_func_isnottrue() = default;
-  virtual const char* func_name() const { return "isnottrue"; }
-  bool find_not_null_fields(table_map allowed) { return false; }
-  Item *get_copy(THD *thd)
+  virtual const char* func_name() const override { return "isnottrue"; }
+  bool find_not_null_fields(table_map allowed) override { return false; }
```

Comment:
While we are changing the lines, we could also fix any `-Wunused-parameter` warnings:
```c++
  bool find_not_null_fields(table_map) override { return false; }
```

Reply:
Agreed that these should be fixed but in a different patch.